tests/bluetooth/bsim_bt/compile.sh

```diff
@@ -20,6 +20,8 @@ mkdir -p ${WORK_DIR}
 
 source ${ZEPHYR_BASE}/tests/bluetooth/bsim_bt/compile.source
 
+app=tests/bluetooth/bsim_bt/bsim_test_host conf_file=prj_eatt.conf compile
```

Comment:
can we use the module name (att, gatt, l2cap, etc), rather than such a big name, i.e. host?

Reply:
Yes, we need to decide on a structure for the bsim tests.

Reply:
I think also the EATT connection test can be combined with the L2CAP collision mitigation test, as they do almost the same. The collision mitigation test must also be updated to use the connected callbacks instead of sleeping.